python_files/supplementary_plot_helpers.py

```diff
@@ -19,6 +20,13 @@
     11, 12, 13, 14, 15, 17, 18, 20, 22, 23, 24, 28, 29, 30, 31, 32, 33, 34, 35,
     36, 39, 40, 41, 42, 43, 44, 45, 46, 47
 ]
+TIMEPOINT_INDICES = ["primary", "baseline", "post_induction", "on_nivo"]
```

Comment:
This should be primary_untreated, that's the main primary timepoint we've been working with